### PR TITLE
added tests for ShoppingCart, User, Order and OrderItem functionality

### PR DESCRIPTION
**Pull request by c-kalenyk**
Снимок экрана (339)
Снимок экрана (340)

### CODE REVIEW

File: src/main/resources/db/changelog/changes/14-add-precision-to-total-in-orders-table.yaml [hunk 1]
```diff
@@ -0,0 +1,9 @@
+databaseChangeLog:
+  - changeSet:
+      id: add-precision-to-total-in-orders-table
+      author: oleksandr
+      changes:
+        - modifyDataType:
```
Comment (Elena-Bruyako) [inline src/main/resources/db/changelog/changes/14-add-precision-to-total-in-orders-table.yaml]: auto rollback not supporter for such operation
remove this changeset and change data type in exists changelog where you create orders

**Review by Elena-Bruyako — CHANGES_REQUESTED** [review]
one comment

File: src/main/resources/db/changelog/changes/14-add-precision-to-total-in-orders-table.yaml (hunk 1 above)
Comment (c-kalenyk) [reply src/main/resources/db/changelog/changes/14-add-precision-to-total-in-orders-table.yaml]: gtk, i thought that any changes to the existing scripts will lead to failure

Comment (Elena-Bruyako) [reply src/main/resources/db/changelog/changes/14-add-precision-to-total-in-orders-table.yaml]: Correct, if you change existing changelogs and run app liquibase will fail, that's why you need to drop db. 
If you would like to modify data type you need to implement correct rollback

**Review by Elena-Bruyako — CHANGES_REQUESTED** [review]
See comment

**Review by okuzan — APPROVED** [review]

**Pull request merged**